pkg/core/proto/protocol.proto

```diff
@@ -1,17 +1,37 @@
 syntax = "proto3";
 
-import "google/protobuf/timestamp.proto";
-
 package protocol;
 
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+
 option go_package = "./proto;proto";
 
 service Protocol {
-  rpc SendTransaction (SendTransactionRequest) returns (TransactionResponse);
-  rpc GetTransaction (GetTransactionRequest) returns (TransactionResponse);
-  rpc GetBlock(GetBlockRequest) returns (BlockResponse);
-  rpc GetNodeInfo(GetNodeInfoRequest) returns (NodeInfoResponse);
-  rpc Ping (PingRequest) returns (PingResponse);
+  rpc SendTransaction(SendTransactionRequest) returns (TransactionResponse) {
+    option (google.api.http) = {
+      post: "/core/grpc/transaction"
+      body: "transaction"
+    };
+  }
+  rpc ForwardTransaction(ForwardTransactionRequest) returns (ForwardTransactionResponse) {
+    option (google.api.http) = {
+      post: "/core/grpc/transaction/forward"
+      body: "transaction"
+    };
```

Comment:
these rest annotations are what become the generated protocol.swagger.json which in turn you can generate clients on top of